src/tree/FunctionAppProvider.ts

```diff
@@ -68,22 +68,22 @@ export class FunctionAppProvider implements IChildProvider {
 
 async function getDefaultRuntime(actionContext: IActionContext): Promise<ProjectRuntime> {
     // Try to get VS Code setting for runtime (aka if they have a project open)
-    let runtime: ProjectRuntime | undefined = getFuncExtensionSetting(projectRuntimeSetting);
+    let runtime: string | undefined = getFuncExtensionSetting(projectRuntimeSetting);
     actionContext.properties.runtimeSource = 'VSCodeSetting';
 
-    if (runtime === undefined) {
+    if (!runtime) {
```

Comment:
Is this the same behavior you intend to have for 0, null and other false-y values?

Reply:
Yep